### ISSUE TITLE
sourceType filter not working in sc.analysis.vuln

### ISSUE THREAD

**Issue by CyberTeam4321** [tenable/pyTenable#606]
**Describe the bug**
If I include sourceType in filters, then pyTenable throws an error

    [403: POST] https://ipaddress:443/rest/analysis body=b'{"type":"regular","response":"","error_code":146,"error_msg":"The filter \'t\' is invalid (valid filters: acceptedRisk, acceptRiskStatus, assetID, auditFileID, benchmarkName, cceID, cpe, cveID, baseCVSSScore, cvssVector, cvssV3BaseScore, cvssV3Vector, dataFormat, daysMitigated, daysToMitigated, dnsName, exploitAvailable, exploitFrameworks, familyID, firstSeen, iavmID, ip, lastMitigated, lastSeen, mitigatedStatus, msbulletinID, outputAssets, patchPublished, pluginModified, pluginPublished, pluginID, pluginName, pluginText, pluginType, policyID, port, protocol, recastRisk, recastRiskStatus, repositoryIDs, responsibleUserIDs, severity, solutionID, stigSeverity, tcpport, udpport, uuid, vulnPublished, wasMitigated, xref, vprScore, assetCriticalityRating).\\n","warnings":[],"timestamp":1660041733}\n'
Traceback (most recent call last):
  File "c:\Users\cyb\Documents\Project\testing1.py", line 6, in <module>
    next(vulns)
  File "C:\Users\cyb\AppData\Local\Programs\Python\Python310\lib\site-packages\tenable\base\v1.py", line 76, in __next__
    return self.next()
  File "C:\Users\cyb\AppData\Local\Programs\Python\Python310\lib\site-packages\tenable\base\v1.py", line 91, in next
    self._get_page()
  File "C:\Users\cyb\AppData\Local\Programs\Python\Python310\lib\site-packages\tenable\sc\analysis.py", line 101, in _get_page
    resp = self._api.post('analysis', json=query).json()
  File "C:\Users\cyb\AppData\Local\Programs\Python\Python310\lib\site-packages\restfly\session.py", line 631, in post
    return self._req('POST', path, **kwargs)
  File "C:\Users\cyb\AppData\Local\Programs\Python\Python310\lib\site-packages\restfly\session.py", line 559, in _req
    raise error_resp
restfly.errors.APIError: [403: POST] https://ipaddress:443/rest/analysis body=b'{"type":"regular","response":"","error_code":146,"error_msg":"The filter \'t\' is invalid (valid filters: acceptedRisk, acceptRiskStatus, assetID, auditFileID, benchmarkName, cceID, cpe, cveID, baseCVSSScore, cvssVector, cvssV3BaseScore, cvssV3Vector, dataFormat, daysMitigated, daysToMitigated, dnsName, exploitAvailable, exploitFrameworks, familyID, firstSeen, iavmID, ip, lastMitigated, lastSeen, mitigatedStatus, msbulletinID, outputAssets, patchPublished, pluginModified, pluginPublished, pluginID, pluginName, pluginText, pluginType, policyID, port, protocol, recastRisk, recastRiskStatus, repositoryIDs, responsibleUserIDs, severity, solutionID, stigSeverity, tcpport, udpport, uuid, vulnPublished, wasMitigated, xref, vprScore, assetCriticalityRating).\\n","warnings":[],"timestamp":1660041733}\n'

**To Reproduce**
from tenable.sc import TenableSC
sc = TenableSC(<ipaddress>)
sc.login(access_key=<access_key_value>, secret_key=<secret_key_value>)
payload = {'type': 'vuln', 'sourceType': 'patched'}
vulns = sc.analysis.vulns(*payload)
next(vulns)

**Expected behavior**

**Screenshots**

**System Information (please complete the following information):**
 - OS: Windows
 - Architecture 64 bit
 - Version pyTenable==1.4.6
 - Memory 4G

**Additional context**

**Comment by SteveMcGrath** [comment]
sourceType isnt a filter.  use the `source='patched'` keyword like so:

```python
from tenable.sc import TenableSC
with TenableSC(access_key=akey, secret_key=skey) as sc:
    for vuln in sc.analysis.vulns(source='patched'):
        print(vuln)
```

**Comment by CyberTeam4321** [comment]
Hello @SteveMcGrath , I have stated the source as the filter and I am able to get mitigated result however I am see lot of informative vulnerability. I am trying to filter the result to based on severity which values are Low, Medium, High and Critical.
I implemented the new Python code

for vuln in sc.analysis.vulns(source='patched',severity!=0):
    print(vuln)
sc.logout()

It mention an errors below.

SyntaxError: expected ':'
